sdk/bcs/package.json

```diff
@@ -59,5 +59,8 @@
     "tsup": "^6.5.0",
     "typescript": "^4.9.4",
     "vitest": "^0.26.3"
+  },
+  "dependencies": {
+    "base58-js": "^1.0.5"
```

Comment:
We use `bs58` in the SDK, can we consolidate onto one or the other? `bs58` seems to have types and is used in some other crypto projects, so minor preference towards that

Reply:
Oh, I should've checked what we use. Thanks for the tip!